examples/game/breakout.rs

```diff
@@ -1,16 +1,53 @@
+//! A simplified implementation of the classic game "Breakout"
+
 use bevy::{
     core::FixedTimestep,
+    math::{const_vec2, const_vec3},
     prelude::*,
     sprite::collide_aabb::{collide, Collision},
 };
 
-/// An implementation of the classic game "Breakout"
+// Defines the amount of time that should elapse between each physics step.
 const TIME_STEP: f32 = 1.0 / 60.0;
+
+// These constants are defined in `Transform` units.
+// Using the default 2D camera they correspond 1:1 with screen pixels.
+// The `const_vec3!` macros are needed as functions that operate on floats cannot be constant in Rust.
+const PADDLE_HEIGHT: f32 = -215.0;
```

Comment:
the meaning of this one is not very obvious, even less with the `PADDLE_SIZE` just under

Reply:
Agreed. I'll fix this in a follow-up PR.